app/views/embed/iframe.html.erb

```diff
@@ -20,7 +20,7 @@
     <%= javascript_importmap_tags @embed_response.viewer.importmap if @embed_response.viewer.importmap %>
     <%= stylesheet_link_tag @embed_response.viewer.stylesheet %>
     <%= stylesheet_link_tag 'sul_icons.css' %>
-    <%= tag :link, rel: "up", href: Embed::Purl.find(@embed_response.request.purl_object.collections.first).purl_url if @embed_response && @embed_response.request.purl_object.collections.any? %>
+    <%= tag :link, rel: "up", href: @embed_response.request.purl_object.first_collection_url if @embed_response && @embed_response.request.purl_object.first_collection_url %>
```

Comment:
If a `Purl` instance has no collections, won't `Purl#first_collection_url` `return "#{Settings.purl_url}/#{nil}"` (truthy)?

Reply:
This is now handled in the model. Thank you.

Reply:
Excellent. :shipit: